sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/api/Azure.ResourceManager.DeviceUpdate.netstandard2.0.cs

```diff
@@ -442,6 +443,12 @@ public partial class DiagnosticStorageProperties
         public string ConnectionString { get { throw null; } set { } }
         public string ResourceId { get { throw null; } set { } }
     }
+    public partial class Encryption
+    {
+        public Encryption() { }
+        public System.Uri KeyVaultKeyUri { get { throw null; } set { } }
+        public string UserAssignedIdentity { get { throw null; } set { } }
```

Comment:
I believe this should be the arm resource id for a user assigned identity resource is expected here? Could you please confirm this?
If so, we could change the format of this property to `arm-id` so that it is more specific.

Reply:
Yes, it is arm-id of the UA MSI. Can the change happen in SDK, not in swagger?

Reply:
Made the change as requested.